### PR TITLE
Added note to The Length Property.

### PR DESCRIPTION
**Pull request by itsJuicyBytes**
Pointed people to go to a different section to do the resizing of the array.

### CODE REVIEW

File: lectures/140_arrays/readme.md [hunk 1]
```diff
@@ -208,6 +208,22 @@ while (counter2 < customArray.Length)
...
+test.Length = 9;
+```
+
+would return, at compile time,
+
+```
+Compilation error (line 8, col 3): Property or indexer 'System.Array.Length' cannot be assigned to --it is read only.
+```
+
+Resizing will be discussed in the section: **Changing the Size**.
```
Comment (aubertc) [inline lectures/140_arrays/readme.md:224]: Add an actual link.

File: lectures/140_arrays/readme.md [hunk 2]
```diff
@@ -208,6 +208,22 @@ while (counter2 < customArray.Length)
...
+
+```
+int[] test = new int[10];
+test.Length = 9;
+```
+
+would return, at compile time,
+
+```
+Compilation error (line 8, col 3): Property or indexer 'System.Array.Length' cannot be assigned to --it is read only.
```
Comment (aubertc) [inline lectures/140_arrays/readme.md:221]: explain what read only means.

**Comment by aubertc** [comment]
Can you improve, please, @itsJuicyBytes .

**Comment by itsJuicyBytes** [comment]
Added the link and added an explanation to read only.

**Comment by aubertc** [comment]
Thanks!

**Pull request merged**